manifests/ovs_netattachdef/netattachdef.yaml

```diff
@@ -15,5 +15,8 @@ spec:
 {{end}}{{if (index . "SelectiveVlan") }}
     "trunk": {{.SelectiveVlan}},
 {{end}}
-    "bridge": "{{.NetObjName}}"
+    "bridge": "{{.NetObjName}}",
```

Comment:
Here if an IPAM is not defined we will have a hanging comma at the end of line 18 with nothing to follow after that. Can we fix this ?

Reply:
This would be fixed as part of stitching.

Reply:
OK. I worry mostly for functionality problem when this NAD is consumed from a pod. MAybe this extra comma will cause problems. I will test though

Reply:
Yes if there is no ipam this would fail,will push temporary change so that it wont cause a problem.

Reply:
ok